samza-core/src/main/java/org/apache/samza/zk/ZkJobCoordinator.java

```diff
@@ -76,44 +75,33 @@
   private JobModel newJobModel;
   private int debounceTimeMs;
 
-  public ZkJobCoordinator(Config config, MetricsRegistry metricsRegistry) {
+  ZkJobCoordinator(Config config, MetricsRegistry metricsRegistry, ZkUtils zkUtils) {
```

Comment:
Thanks for cleaning up the constructor and moving the relevant parts to the factory. 

QQ: Why has the scope changed from `public` ?